Plugins/Damage/Damage.cpp

```diff
@@ -211,6 +213,7 @@ NWNX_EXPORT ArgumentStack SetAttackEventData(ArgumentStack&& args)
         k = (int16_t)args.extract<int32_t>();
     }
     s_AttackData.nAttackResult = args.extract<int32_t>();
+    s_AttackData.nSneakAttack = args.extract<int32_t>();
```

Comment:
This is a breaking ABI change, scripts compiled with the old version will not send this last var and it'll throw a runtime error. 

Since we're currently on a development build I think it's ok, but you should mention it in the changelog with a big ol' `BREAKING CHANGE`

Reply:
ABI breaking note added to the changelog.